inference-engine/src/mkldnn_plugin/CMakeLists.txt

```diff
@@ -207,6 +207,10 @@ set_ie_threading_interface_for(${TARGET_NAME}_obj)
 target_compile_definitions(${TARGET_NAME}_obj PUBLIC -DMKLDNN_THR=${MKLDNN_THR}
                                               PRIVATE USE_STATIC_IE IMPLEMENT_INFERENCE_ENGINE_PLUGIN)
 
+if (NOT BUILD_SHARED_LIBS)
+    target_compile_definitions(${TARGET_NAME}_obj PRIVATE NGRAPH_STATIC_LIBRARY)
+endif()
```

Comment:
maybe we can add `$<TARGET_PROPERTY:ngraph,INTERFACE_INCLUDE_DIRECTORIES>` to `target_include_directories` above?

Reply:
Will that add the compile definition ?
By the way i was thinking about renaming NGRAPH_STATIC_LIBRARY to USE_STATIC_NGRAPH